### ISSUE TITLE
Issues with dtreeviz library dependancy

### ISSUE THREAD

**Issue by suraj-gade** [oegedijk/explainerdashboard#253]
Hi, 
I am getting below error while importing the latest version of explainer dashboard with Python 3.9.13
`from explainerdashboard import ClassifierExplainer, ExplainerDashboard` for the detreeviz library

Error:
```
from dtreeviz.trees import ShadowDecTree, dtreeviz
ImportError: cannot import name 'dtreeviz' from 'dtreeviz.trees'
```

I have a working deployment of ML pipeline which uses explainer dashboard library's older version 0.3.8 and Python 3.7 

So, tried using older version of explainer dashboard `explainerdashboard==0.3.8` and downgrading Python to 3.7 but it didn't help. 
I tried installing dtreeviz separately as well, but issue is still persistent.

Please help me with this and let me know if I am missing something.

**Comment by catyung** [comment]
Hi, 

Can you try to downgrade dtreeviz to 1.4.1 

pip install dtreeviz==1.4.1

**Comment by oegedijk** [comment]
Yeah, this is an issue with the new dtreeviz 2.0 api, will see if I have some time this weekend to push a fix...

**Comment by oegedijk** [comment]
Okay, just released v0.4.2 which should force update to dtreeviz>2.1 and then uses the new API. So I think that should fix the issue.

Let me know if it worked for you @suraj-gade
